app/controllers/reviews_controller.rb

```diff
@@ -0,0 +1,19 @@
+class ReviewsController < ApplicationController
+
+  def new
+    @shelter = Shelter.find(params[:shelter_id])
+  end
+
+  def create
+    shelter = Shelter.find(params[:shelter_id])
+    shelter.reviews.create!(review_params)
```

Comment:
I was under the impression that these bang operators were only used inside of tests & not in code that can reach production. Will get final confirmation but this should not hold up the PR.

Reply:
I agree, that was my understanding as well.  Maybe Jesse can go ahead and remove those.  Leave the ones in the test, remove the ones in the code.

Reply:
Cool. Just remember that the .save! needs to be added on the next line if you do that... the syntax is in the handling lesson.

Reply:
create! changed to create
not using new/save